src/com/javabasic/lesson4/FoodForPets.java

```diff
@@ -0,0 +1,64 @@
+package com.javabasic.lesson4;
+
+import java.text.DecimalFormat;
+import java.util.Scanner;
+
+public class FoodForPets {
+
+    public static void main(String[] args) {
+
+        DecimalFormat decimalFormat = new DecimalFormat( "#.###" );
+
+        Scanner scanner = new Scanner(System.in);
+
+        float oneFoodPortionWeight = 0.05F;
+        int oneFoodPortionCoast = 2;
+        int thirtyDaysMonth = 30;
+        int numberOfMeals = 3;
+
+        System.out.println("Please input numbers of Dogs and Cats:");
+        int countOfDogs = scanner.nextInt();
+        int countOfCats = scanner.nextInt();
+
+        float monthlyFoodWeightForCats = monthFoodWeightForPets(
+                countOfCats, thirtyDaysMonth, numberOfMeals, oneFoodPortionWeight
+        );
+
+        float monthlyFoodWeightForDogs = monthFoodWeightForPets(
+                countOfDogs, thirtyDaysMonth, numberOfMeals, oneFoodPortionWeight
+        );
+
+        System.out.printf(
+                "For %s days: %s Cats need = %s kilo, %s Dogs need = %s kilo of food.%n",
+                thirtyDaysMonth,
+                countOfCats,
+                decimalFormat.format(monthlyFoodWeightForCats),
+                countOfDogs,
+                decimalFormat.format(monthlyFoodWeightForDogs)
+        );
+
+        System.out.printf(
+                "For %s days, for all your pets, you need = %.2f dollars",
+                thirtyDaysMonth,
+                monthlyCoastForPetFood(oneFoodPortionCoast, monthlyFoodWeightForDogs,monthlyFoodWeightForCats)
+        );
+
+    }
+
+    public static float monthFoodWeightForPets(int count, int days, int numberOfMeals, float weight){
+       return weight * count * numberOfMeals * days;
+    }
+
+    public static float monthlyCoastForPetFood(
```

Comment:
This method will be hard to extend if we add new pets.
Another point, we have code duplication in this method for each pet type.
Maybe it's better to use the same approach as `monthFoodWeightForPets `
Calculate the cost for only one pet type and summarize outside this method.
What do you think?

Reply:
Totally agree with you. Made some changes, could you please check them?